src/sass/index.sass

```diff
@@ -283,9 +292,18 @@
               .filter-item
                 padding-top: 1rem
 
-        .sheet-body
-          overflow-y: auto
-          border-top: 1px solid #69675d
+      .sheet-body
+        overflow-y: auto !important
```

Comment:
Needed an !important here because long item/spell descriptions in popups wouldn't allow scrolling otherwise